meta.go

```diff
@@ -0,0 +1,14 @@
+package azuretpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "azure.cluster.giantswarm.io"
```

Comment:
@kopiczko I remember you wanting to change this so the TPRs are in `giantswarm.io`. Is that still the case?